packages/providers/mgt-msal2-provider/src/Msal2Provider.ts

```diff
@@ -532,10 +532,10 @@ export class Msal2Provider extends IProvider {
   public getActiveAccount() {
     const account = this._publicClientApplication.getActiveAccount();
     return {
-      name: account.name,
-      mail: account.username,
-      id: account.homeAccountId,
-      tenantId: account.tenantId
+      name: account?.name,
```

Comment:
TBH I'd do one nullish check and return undefinded in the nullish case rather than an object with potentially undefined keys. also resulting JavaScript when transpiled to es2019 or lower (SPFx scenarios) will be much more efficient.